Changing Permissions and Techical Contact (#6128)
mendix/docs: content/en/docs/apidocs-mxsdk/apidocs/deploy-api-3.md

### 3.2 Using the API to Change App Team Members' Access Permissions to an Environment

The following steps will change the permissions of a team member to an environment of the app identified by the UUID {appId}:

> {{% alert color="info" %}}Only the following people can change the access permissions of team members: Mendix Admins of the company, the Technical Contact of the app, and the app team members with specific permissions to manage permissions.{{% /alert %}}

Reviewer: Perhaps simpler to say
```suggestion
{{% alert color="info" %}}Only the following people can change the access permissions of team members: Mendix Admins of the company, the Technical Contact of the app, and the app team members who can manage permissions.{{% /alert %}}
```
although slightly less precise technically?
